lib/ExpensiMark.js

```diff
@@ -110,12 +110,12 @@ export default class ExpensiMark {
              */
             {
                 name: 'userMentions',
-                regex: new RegExp(`[a-zA-Z0-9.!$%&+/=?^\`{|}-]?@+${CONST.REG_EXP.EMAIL_PART}(?!((?:(?!<a).)+)?<\\/a>|[^<]*(<\\/pre>|<\\/code>))`, 'gm'),
+                regex: new RegExp(`[a-zA-Z0-9.!$%&+/=?^\`{|}-]?@+${CONST.REG_EXP.EMAIL_PART}(?!((?:(?!<a).)+)?<\\/a>|[^<]*(<\\/pre>|<\\/code>))`, 'gmi'),
                 replacement: (match) => {
                     if (!Str.isValidMention(match)) {
                         return match;
                     }
-                    return `<mention-user>${match}</mention-user>`;
+                    return `<mention-user>${match.toLowerCase()}</mention-user>`;
```

Comment:
@nikhildewoolkar Won't this convert the input to lowercase? As discussed in the issue, we don't want to convert the input to lower case. We want to leave it as it is.

Context: https://github.com/Expensify/App/issues/26016#issuecomment-1706767335
https://github.com/Expensify/App/issues/26016#issuecomment-1713320051

Reply:
Ok yes good catch, thank you. I agree that we don't want to make it lowercase here.

That also makes me think that we should have tests for converting a mention in html back to markdown, with uppercase, lowercase, mixed, etc.